dockercheck

```diff
@@ -1,52 +1,78 @@
 #!/usr/bin/python3
 
 import os
+import psutil
 import logging
 import threading
 import cherrypy
 import docker
 from prometheus_client import generate_latest, Enum
 
 try:
-    LISTEN_PORT = int(os.environ.get("DOCKERCHECK_HTTP_PORT","8080"))
+    LISTEN_PORT = int(os.environ.get("DOCKERCHECK_HTTP_PORT", "8080"))
 except:
     logging.error("DOCKERCHECK_HTTP_PORT must be an integer value")
     exit(2)
 
-DOCKER_LABEL = os.environ.get("DOCKERCHECK_LABEL","com.docker.compose.project")
-METRIC_NAME  = os.environ.get("DOCKERCHECK_PROM_METRIC",'container_status')
+DOCKER_LABEL = os.environ.get("DOCKERCHECK_LABEL", "com.docker.compose.project")
+METRIC_NAME = os.environ.get("DOCKERCHECK_PROM_METRIC", "container_status")
+
+prom_container_status = Enum(
+    METRIC_NAME,
+    "Returns the status of a Compose project container",
+    states=["restarting", "running", "paused", "exited", "unhealthy"],
+    labelnames=["name", "role"],
+)
 
-prom_container_status = Enum(METRIC_NAME, 'Returns the status of a Compose project container',
-        states=['restarting', 'running', 'paused', 'exited', 'unhealthy'], labelnames=["name", "role"])
 
 def check_containers_status(client, filter_label):
-    '''
-    Check the containers statuses and export metrics
+    """
+    Check the containers statuses and disk usage and export metrics
 
     Iterates over a list of containers in *any* state and filtering on `filter_label`
-    '''
-    containers = client.containers.list(all=True,
-                                        filters={"label": filter_label})
+    Updates Prometheus metrics for each container and logs disk usage errors.
+
+    It returns HTTP status code (200) if all containers are running and disk usage is below 90%.
+
+    """
+    containers = client.containers.list(all=True, filters={"label": filter_label})
     if not containers:
-        return(500)
+        return 500
 
-    http_status=200
+    http_status = 200
     for container in containers:
-        prom_container_status.labels(container.name,container.labels[filter_label]).state(container.status)
+        prom_container_status.labels(
+            container.name, container.labels[filter_label]
+        ).state(container.status)
         if container.status != "running":
-            http_status=500
+            http_status = 500
         try:
             if container.attrs["State"]["Health"]["Status"] == "unhealthy":
                 # health can be "starting", "healthy" and "unhealthy". we care just about the latter
-                prom_container_status.labels(container.name,container.labels[filter_label]).state("unhealthy")
-                http_status=500
+                prom_container_status.labels(
+                    container.name, container.labels[filter_label]
+                ).state("unhealthy")
+                http_status = 500
         except KeyError:
             pass
-    return(http_status)
+    disk_usage_percent = psutil.disk_usage("/").percent
```

Comment:
Just to be defensive I would put this in an try block, and always return `http_status = 500` in case there is any exception here